redis/asyncio/connection.py

```diff
@@ -1646,7 +1652,10 @@ async def disconnect(self, inuse_connections: bool = True):
         self._checkpid()
         async with self._lock:
             resp = await asyncio.gather(
-                *(connection.disconnect() for connection in self._connections),
```

Comment:
This disconnects only those connections _in_ the pool, not _all_ connections as before.